src/python/CMakeLists.txt

```diff
@@ -1,54 +1,29 @@
-#if the CMAKE_CURRENT_BINARY_DIR is python3, then build the python3 module,
-#otherwise the python2 module
-string (REGEX MATCH "python3\$" _py3_subdir ${CMAKE_CURRENT_BINARY_DIR})
-
-if (_py3_subdir)
-    set (BUILD_PY3 ON)
-else ()
-    set (BUILD_PY3 OFF)
+# Narrow the search for python to only those supported
+if (NOT DEFINED Python_ADDITIONAL_VERSIONS)
+    # Support newer versions first. Can be overridden.
+    set (Python_ADDITIONAL_VERSIONS "3.7" "3.6" "3.5" "3.4" "2.7")
```

Comment:
I'm not 100% positive that this is needed. These values are default in the find package anyways. Also, I think I may need to add CACHE STRING to it so that it shows up when listing all user settable options, then it is know that the python version range can be set manually.

Reply:
Let's take it out, then, and see what happens.